pkg/cli/start.go

```diff
@@ -343,7 +343,7 @@ func runStart(cmd *cobra.Command, args []string) error {
 			var err error
 			s, err = server.NewServer(serverCfg, stopper)
 			if err != nil {
-				return errors.Wrap(err, "failed to start Cockroach server")
+				return errors.Wrap(err, "failed to start CockroachDB server")
```

Comment:
I don't think this message should depend on the binary name. In the interest of not bike shedding about this, are you OK just dropping the word `Cockroach` instead? It doesn't add anything - you know what you're trying to run:

```
$ ./cockroach start
*
* ERROR: failed to start Cockroach server: problem using security settings, did you mean to use --insecure?: no CA certificate found
*
Failed running "start"
```